build_tools/cmake/external_cc_library.cmake

```diff
@@ -85,106 +85,109 @@ function(external_cc_library)
     ${ARGN}
   )
 
-  if(NOT _RULE_TESTONLY OR IREE_BUILD_TESTS)
-    # Prefix the library with the package name.
-    string(REPLACE "::" "_" _PACKAGE_NAME ${_RULE_PACKAGE})
-    set(_NAME "${_PACKAGE_NAME}_${_RULE_NAME}")
+  # Return early if this is test only but we are not building tests.
```

Comment:
nit: comment is redundant, the code is easy enough to read
(in all files)

Reply:
done